### PR TITLE
fix(conf): can't include local/overide-conf file

### PR DESCRIPTION
**Pull request by zhongwencool**
the inlcude_dirs should be a dir list, not filename list.

### CODE REVIEW

File: apps/emqx/src/emqx_config.erl [hunk 1]
```diff
@@ -280,7 +281,7 @@ init_load(SchemaMod, RawConf) when is_map(RawConf) ->
                             maps:with(RootNames, RawConfWithEnvs)).
 
 include_dirs() ->
-    [filename:join(emqx:data_dir(), "configs")].
+    [filename:join(emqx:data_dir(), "configs") ++ "/"].
```
Comment (terry-xiaoyu) [inline apps/emqx/src/emqx_config.erl:284]: Is this an issue of hocon_schema? If so we may also need to fix it in the hocon lib.

Comment (zhongwencool) [reply apps/emqx/src/emqx_config.erl:284]: Fix on https://github.com/emqx/hocon/pull/166

Comment (zmstone) [inline apps/emqx/src/emqx_config.erl:284]: is this still needed ?
e.g. in windows, the path will become `prefix/\name` after `filename:join`

Comment (zhongwencool) [reply apps/emqx/src/emqx_config.erl:284]: OH, forget windows. 
we don't need  this  now !

**Review by zmstone — APPROVED** [review]

**Pull request merged**